### DISCUSSION TITLE
Is it possible to change the font size for node labels?

### DISCUSSION THREAD

**Discussion by ab-be-old** [reaviz/reagraph#178] [Q&A]
There's the fontSize parameter for edges but no equivalent for nodes. Did I miss something in the documentation, or is it planned in the future? Currently the default node label size is very small for a larger graph and I did not succeed in changing it, any workaround would be appreciated.
Thanks a lot!

**Comment by manishkc104** [comment]
@ab-be-old did you get fix this issue on how to resize the label of the Node ?

**Reply by amcdnl** [reply]
Happy to accept a PR for this.
